flutter/lib/ui/settings/settings_screen.dart

```diff
@@ -199,6 +201,11 @@ class _SettingsScreen extends State<SettingsScreen> {
             },
             child: Text(stringResources.settingsClearCache),
           ),
+          const Divider(),
+          Text(
+            'Version: ${buildInfo.version} | Build: ${buildInfo.buildNumber}',
```

Comment:
On my phone this text is obstructed by the control button at the bottom of the screen.
<img src="https://user-images.githubusercontent.com/36223296/193507671-5a0d18a6-4981-43c5-b3c8-86bbaa2e73b5.jpg" width=250>

You can make the `ListView` a child of `SafeArea` element to fix it.
https://github.com/mlcommons/mobile_app_open/blob/d76773f447d6f385499dca578c06a7e4743f8a8c/flutter/lib/ui/settings/settings_screen.dart#L56